docs/concepts/storage/persistent-volumes.md

```diff
@@ -108,6 +108,8 @@ The Retain reclaim policy allows for manual reclamation of the resource. When th
 
 #### Recycling
 
+**Note that the recycyling reclaim policy is being deprecated. Instead, the recommended approach is to use dynamic provisioning.**
```

Comment:
Please refer to https://kubernetes.io/docs/home/contribute/style-guide/#callout-formatting for the correct formatting of callouts.

Also please fix the typo: recycling

Reply:
When things are deprecated, better clearly state the version number when it is deprecated.

Reply:
@tengqm It's not exactly clear to me in which release this will be deprecated. The announcement was made here https://groups.google.com/forum/#!topic/kubernetes-dev/uexugCza84I. Probably @saad-ali can help.